src/zcash/Zcash.h

```diff
@@ -28,4 +28,13 @@
 #define ZC_SAPLING_ENCCIPHERTEXT_SIZE (ZC_SAPLING_ENCPLAINTEXT_SIZE + NOTEENCRYPTION_AUTH_BYTES)
 #define ZC_SAPLING_OUTCIPHERTEXT_SIZE (ZC_SAPLING_OUTPLAINTEXT_SIZE + NOTEENCRYPTION_AUTH_BYTES)
 
+// ZIP 225 transaction sizes (V5 transaction format)
+#define ZC_ZIP225_BASE_TRANSACTION_SIZE 108
+#define ZC_ZIP225_BASE_ORCHARD_PROOF_SIZE 2720
+#define ZC_ZIP225_BASE_SIZE (ZC_ZIP225_BASE_TRANSACTION_SIZE + ZC_ZIP225_BASE_ORCHARD_PROOF_SIZE)
+// Variable size calculations are ultimately based on the number of nActionsOrchard sections
+#define ZC_ZIP225_VARIABLE_TRANSACTION_SIZE 884
+#define ZC_ZIP225_VARIABLE_ORCHARD_PROOF_SIZE 2272
+#define ZC_ZIP225_VARIABLE_SIZE (ZC_ZIP225_VARIABLE_TRANSACTION_SIZE + ZC_ZIP225_VARIABLE_ORCHARD_PROOF_SIZE)
```

Comment:
I don't know how to verify these constants. I looked at [zip225](https://zips.z.cash/zip-0225) but it wasn't obvious (but maybe I just didn't look carefully enough). If it's possible to add a comment that cites a reference, that would be nice.

Reply:
Force-pushed to address review comments